subscription_manager/controller.py

```diff
@@ -10,12 +11,12 @@ def __init__(self, dbhelper: DBHelper):
         """Takes DBHelper instance for communicating with database"""
         self.dbhelper = dbhelper
 
-    def add_subscription(self, subscription: dict) -> int:
+    def add_subscription(self, subscription_dict: dict) -> int:
```

Comment:
I suppose it should receive all the dict keys and process Subscription object to DBhelper

Reply:
done